tests/kernel/install_ltp.pm

```diff
@@ -31,7 +31,10 @@ sub scc_we_enabled {
 
 sub add_desktop_productivity_module {
     if (get_required_var('ARCH') eq 'x86_64' and check_var('DISTRI', 'sle') and sle_version_at_least('15')) {
-        add_suseconnect_product("sle-module-desktop-productivity");
+        zypper_call(
+            'ar http://openqa.suse.de/assets/repo/SLE-15-Module-Desktop-Productivity-POOL-x86_64-Build' . get_required_var('BUILD_SLE') . '-Media1/ DPM',
```

Comment:
Could you please use REPO_SLE15_MODULE_DESKTOP_PRODUCTIVITY variable here, then we are more scalable

Reply:
@rwx788 sure, good point. Fixed.